incubator/binding-grpc/src/test/resources/proto/echo.proto

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+syntax = "proto3";
+
+package io.aklivity.zilla.runtime.binding.grpc.internal;
+
+option java_multiple_files = true;
+option java_outer_classname = "EchoProto";
+
+service EchoService {
+
+  rpc EchoUnary(EchoMessage) returns (EchoMessage) {
+  }
+
+  rpc EchoStream(stream EchoMessage) returns (stream EchoMessage) {
+  }
+}
+
+message EchoMessage {
+  string message = 1;
+}
+
```

Comment:
Do we need a separate copy of this proto here?
Can we use the one from spec, stored relative to the config from spec instead?